<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeekSpaceWindow</class>
 <widget class="QMainWindow" name="GeekSpaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>623</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GeekSpace</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/GeekSpace</normalon>
   </iconset>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>881</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeModules">
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>false</bool>
       </property>
       <column>
        <property name="text">
         <string>Объекты</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuObjects">
    <property name="title">
     <string>Объекты</string>
    </property>
    <widget class="QMenu" name="menu_10">
     <property name="title">
      <string>Добавить</string>
     </property>
     <addaction name="actionAddSignalTimer"/>
     <addaction name="actionAddSignalSheldule"/>
     <addaction name="actionAddProcess"/>
     <addaction name="actionScript"/>
     <addaction name="actionFindModule"/>
    </widget>
    <addaction name="menu_10"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Окно</string>
    </property>
    <addaction name="actionConsole"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuObjects"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockConsole">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>100</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Консоль</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QTextBrowser" name="console">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1091</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionAddSignalSheldule"/>
   <addaction name="actionAddSignalTimer"/>
   <addaction name="actionAddProcess"/>
   <addaction name="actionScript"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normalon>:/exit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFindModule">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/addModule</normaloff>:/addModule</iconset>
   </property>
   <property name="text">
    <string>Модуль</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>Shift+F1</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Консоль</string>
   </property>
  </action>
  <action name="actionAddSignalTimer">
   <property name="icon">
    <iconset>
     <normalon>:/addTimer</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Таймер</string>
   </property>
   <property name="toolTip">
    <string>Добавить таймер</string>
   </property>
  </action>
  <action name="actionAddSignalSheldule">
   <property name="icon">
    <iconset>
     <normalon>:/addCalendar</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Расписание</string>
   </property>
   <property name="toolTip">
    <string>Добавить расписание</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionAddProcess">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/addProcess</normaloff>:/addProcess</iconset>
   </property>
   <property name="text">
    <string>Процесс</string>
   </property>
   <property name="toolTip">
    <string>Добавить процесс</string>
   </property>
  </action>
  <action name="actionScript">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/addScript</normaloff>:/addScript</iconset>
   </property>
   <property name="text">
    <string>Сценарий</string>
   </property>
   <property name="toolTip">
    <string>Добавить сценарий</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
